hhs_hosp/delphi_hhs/run.py

```diff
@@ -143,7 +142,7 @@ def smooth_values(df, smoother):
 def pop_proportion(df,geo_mapper):
     """Get the population-proportionate variants as the dataframe val."""
     pop_val=geo_mapper.add_population_column(df, "state_code")
-    df["val"]=df["val"]/pop_val["population"]*100000
+    df["val"]=round(df["val"]/pop_val["population"]*100000,2)
```

Comment:
we use 7 decimal places systemwide
```suggestion
    df["val"]=round(df["val"]/pop_val["population"]*100000,7)
```